content/TrueCommand/TCGettingStarted/UserAccounts.md

```diff
@@ -8,50 +8,71 @@ weight: 30
 You can create user accounts in the TrueCommand interface. Alternatively, LDAP can automatically create new user accounts when someone logs into TrueCommand with their LDAP credentials.
 
-User accounts also organize into "Teams" for simultaneous management of large numbers or related user accounts.
-
+User accounts also organize into *teams* for simultaneous management of large numbers or related user accounts.
 
 ## Administrator Accounts
 
-TrueCommand has two levels of accounts - Administrators and Users:
+TrueCommand has two levels of accounts - administrators and users:
 
 Administrators can add and remove users and servers.
-Administrators can also assign Users to Teams and Servers to Groups.
-Administrators have full access to all Alerts and Reports.
+Administrators can also assign users to teams and servers to groups.
+Administrators have full access to all alerts and reports.
 
-Users on the other other hand can only interact with the servers they have been assigned by an Administrator.  
+Users on the other other hand can only interact with the servers they are assigned by an administrator.  
 Users can configure alerts and generate reports on their respective systems.
 
+## Users Accounts
 
-## Users and Teams
-
-To create a new user account, open the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> menu and click *Users > + NEW USER*.
+To create a new user account, open the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> menu and click **Users > + NEW USER**.
 Enter a descriptive user name and an authentication method for the user.
 
-TrueCommand uses the *DEFAULT* authentication method to create unique credentials for logging in to the web interface.
-The administrator has to provide these credentials to the intended user.
+TrueCommand uses the *default* authentication method to create unique credentials for logging in to the web interface.
+The administrator must provide these credentials to the intended user.
 
 ![UsersAdd](/images/TrueCommand/2.0/UsersNewUser.png "Adding a new user")
 
+## Two Factor Authentication
+
+
+Two-factor authentication is used to double check the authentication of an account user.
+The first verification occurs when the user logs in with a username and a password.
+Two-factor authentication adds an extra step in the process, a second security layer, that re-confirms their identity.
+If basic password security measures are in place, two-factor authentication makes it more difficult for unverified users to log in to your account.
+
+Enabling Two Factor Authentication requires an email address that has been authenticated. Authenticating a user's email address requires first setting up [SMTP Email]({{<relref "/TrueCommand/alerts/alertmanage.md" >}}) in **Settings -> Alert Services**.
```

Comment:
for consistency do we want to say two-factor authentication or change line 37 to two factor authentication? Also, factor and authentication don't need to be init-capped.

Reply:
two-factor authentication is what I see typically used elsewhere - we should be consistent on that usage.